managing_providers/_topics/adding_azure_cloud_regions.md

```diff
@@ -0,0 +1,31 @@
+# Adding Azure Cloud Regions
+
+{{ site.data.product.title_short }} allows administrators to add additional Azure cloud regions on
+the appliance server. You can use this capability to add new regions that have been
+set up since {{ site.data.product.title_short }} was released.  Once added the region it will be available when
+adding a new Azure provider.
+
+1.  Click ![config gear](../images/config-gear.png) (**Configuration**).
+
+2.  Click on the **Settings** accordion, then click **Zones**.
+
+3.  Click the zone where the {{ site.data.product.title_short }} server is located,
+    then click on the EVM server.
+
+4.  Click on the **Advanced** tab.
+
+5.  Search for `:ems_azure:`, and enter the regions you want to add
+    under `:additional_regions:`.
+
+        Example. To add hypothetical `eastus4` and `eastus4` regions:
```

Comment:
```suggestion
        Example. To add hypothetical `eastus4` and `eastus5` regions:
```

Even so, I like the lunar base example better - can we make these lunar base?

Reply:
As soon as Nadella also starts a personal space program :wink:

Reply:
Done